data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/BeaconRestApi.java

```diff
@@ -105,8 +107,8 @@ private void initialize(
             return jettyServer;
           });
     }
-    app.server().setServerHost(configuration.getRestApiInterface());
-    app.server().setServerPort(configuration.getRestApiPort());
+    Objects.requireNonNull(app.server()).setServerHost(configuration.getRestApiInterface());
+    Objects.requireNonNull(app.server()).setServerPort(configuration.getRestApiPort());
```

Comment:
This is redundant - it will throw a `NullPointerException` with or without `requireNonNull` because that's what `requireNonNull` does when the param is null.  We shouldn't make any change here - the server is known to not be null at this point, just the analyser doesn't know that.

Reply:
Makes sense. Done.